app/src/ui/changes/commit-message.tsx

```diff
@@ -93,6 +93,7 @@ export class CommitMessage extends React.Component<
   ICommitMessageProps,
   ICommitMessageState
 > {
+  private _isMounted: boolean = false
```

Comment:
Hi @ADustyOldMuffin and thanks for looking into this! 

This seems like a reasonable solution, I'd just like to make sure we capture this context for future readers of this code. Could we add a code comment above this with a little context for this and a link to https://reactjs.org/blog/2015/12/16/ismounted-antipattern.html as a reference (i'm assuming that's the article you're referring to in the PR body)? Thank you!

Reply:
Certainly, I added a comment stating that `_isMounted` was being used instead of `this.isMounted()` because the latter is depreciated and then left a link for references as to why.